apis/config/v1beta2/defaults.go

```diff
@@ -26,8 +26,10 @@ import (
 )
 
 var (
-	defaultPermitWaitingTimeSeconds      int64 = 60
-	defaultDeniedPGExpirationTimeSeconds int64 = 20
+	defaultPermitWaitingTimeSeconds        int64 = 60
+	defaultDeniedPGExpirationTimeSeconds   int64 = 20
+	defaultIgnorePodNumCheckingTimeSeconds int64 = 3
```

Comment:
To make the behavior backwards compatible, I think it should be defaulted to 0 (means always check, which is the current behavior) as the default value.